vendor/k8s.io/csi-translation-lib/plugins/openstack_cinder.go

```diff
@@ -54,6 +55,10 @@ func (t *osCinderCSITranslator) TranslateInTreeInlineVolumeToCSI(volume *v1.Volu
 
 	cinderSource := volume.Cinder
 	pv := &v1.PersistentVolume{
+		ObjectMeta: metav1.ObjectMeta{
+			// A.K.A InnerVolumeSpecName required to match for Unmount
+			Name: volume.Name,
+		},
```

Comment:
Why is this in this PR?

Reply:
moved vendor update changes to seperate PR for cleanliness. Also to check for functional changes